doc/src/user_guide.rst

```diff
@@ -1220,6 +1220,14 @@ Plugins
 -------
 
 Plugins allow for powerful additional functionality to be swapped in and out.
+It is possible to load multiple instances of the same plugin by appending an
+integer tag, for example::
```

Comment:
So the tag can be more than an integer.  (Also see the fluidforce plugin which uses the tag explicitly.)